src/pages/o-paralelnej-polis.jsx

```diff
@@ -3,12 +3,87 @@ import React from 'react';
 import { Page } from 'components/Page';
 
 export default () => (
-  <Page title="O Paralelnej Polis">
+  <Page title="O Paralelnej Polis Košice">
+    <blockquote>
+      Paralelná Polis Košice: ostrov slobody a nezávislosti.
+    </blockquote>
+    <h2>Kto sme?</h2>
     <p>
-      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur
-      autem, inventore excepturi eos doloremque reprehenderit provident
-      architecto blanditiis sint nemo quasi at explicabo praesentium. Tempora
-      vero aperiam totam velit consequuntur.
+      Sme kolektív ľudí, ktorí chcú žiť v slobodnejšom a otvorenejšom svete.
+      Hľadáme spôsoby a technológie (Bitcoin, blockchain, reputačné systémy a
+      decentralizované technológie), ktoré nám otvárajú možnosti, zjednodušujú
+      bežné procesy a odstraňujú bariéry. V Paralelnej Polis Košice chceme
+      vytvoriť prostredie zamerané na vzdelávanie, objavovanie a tvorenie
+      lepších systémov pre naše okolie - teba, nás, firmy, jednotlivcov -
+      všetkých, ktorí majú záujem fungovať slobodnejšie a nezávislejšie.
+      Zameriavame sa na experimentovanie a aplikáciu kryptomien a spojených
+      technológií do bežného života.
+    </p>
+    <p>
+      Dnešné technológie a ich exponenciálny rozvoj priniesli veľké výhody ako
+      pre jednotlivcov, tak pre firmy, ktoré sa týkajú hlavne väčšej slobody a
+      otvorenosti, rýchlejších transakcií, nižších poplatkov, decentralizácie.
+      No tieto sú pre väčšinu ľudí ešte neznáme. Naším cieľom je ich priblížiť,
+      vysvetliť a pomôcť uplatniť v praxi. Máme ambíciu byť experimentátormi a
+      inovátormi v týchto perspektívnych oblastiach.
+    </p>
+    <p>
+      Inšpirovali sme sa tímom{' '}
+      <a href="https://paralelnapolis.sk/" target="_blank">
```

Comment:
Please, avoid `target="_blank"`, let the user make the choice

Reply:
In my opinion, it's better to open in new window than to replace the content of the page the user is looking at with what they might not be expecting.

I'd argue that majority of users don't know ctrl+click to open in new tab, or don't do rightclick + open in new tab. 

In this case it isn't that bad, as it's a static page, but this becomes a problem on f.e. contact pages - you fill in a contact form and then decide to click on a link and it reloads the page = you loose your contact form data.

So to summarize - do majority of users know they have a choice? if yes, then no target="_blank" is needed. If less than majority, then i'd leave it there. If we don't know, I'd leave it there. These links are complimentary info linking to external sources, so shouldn't close the page you are looking at + shouldn't replace the content you're looking at - the user might not notice they are on a different website now. Internal links = ok in same window. External links, better in new window. It's lesser evil in my opinion - having a new tab openned, than having the same page reloaded with different content.

But we can polemize about this longer than it's worth it. I don't care that much one way or another if it's a page with no entry fields. If you feel strongly for no new tab, then go ahead and update the code :)

Reply:
Oh my, you wrote such a huge comment so now I feel bad that I'm going to remove those attributes 😭 Sorry!

I don't know whether majority of people do use "open in a new tab" option or not, but with `target="_blank"` we leave no option for those who do. There is no way to force opening window in the same tab if that is desired. (which is also default browsers' action).

Let's just consider our visitors are somehow skilled so we won't use it unless it's really necessary :sunglasses: